lang/rust/avro/Cargo.toml

```diff
@@ -66,7 +66,7 @@ serde_json = { default-features = false, version = "1.0.81", features = ["std"]
 serde = { default-features = false, version = "1.0.137", features = ["derive"] }
 snap = { default-features = false, version = "1.0.5", optional = true }
 strum = { default-features = false, version = "0.24.1" }
-strum_macros = { default-features = false, version = "0.24.1" }
+strum_macros = { default-features = false, version = "0.24.0" }
```

Comment:
On [strum site](https://docs.rs/strum/latest/strum/), they says that 0.24.1 exists but advice to use only "0.24" (i think it's for automatically replace by last minor version ?)